src/vivarium/examples/disease_model/risk.py

```diff
@@ -1,73 +1,112 @@
+from typing import Any, Dict, List
+
 import pandas as pd
 
+from vivarium import Component
+from vivarium.framework.engine import Builder
+
+
+class Risk(Component):
 
-class Risk:
-    configuration_defaults = {
+    CONFIGURATION_DEFAULTS = {
         "risk": {
             "proportion_exposed": 0.3,
         },
     }
 
-    def __init__(self, name):
-        self.name = name
-        self.configuration_defaults = {name: Risk.configuration_defaults["risk"]}
+    ##############
+    # Properties #
+    ##############
+
+    @property
+    def configuration_defaults(self) -> Dict[str, Any]:
+        return {self.risk: self.CONFIGURATION_DEFAULTS["risk"]}
+
+    @property
+    def columns_created(self) -> List[str]:
+        return [f"{self.risk}_propensity"]
+
+    @property
+    def initialization_requirements(self) -> Dict[str, List[str]]:
+        return {
+            "requires_columns": [],
```

Comment:
Don't you need self.columns_created?

Reply:
There isn't a self.columns_created in the Risk class.